<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>713</width>
    <height>373</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>713</width>
    <height>373</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sony Control Center</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>173</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>173</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Battery Charge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Keyboard Backlight</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Touchpad</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ambient Light Sensor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lid Resume Control</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Optical Device</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thermal Profile</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="stackedWidgetPage1" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="chk_battery_fast_charge">
            <property name="text">
             <string>Enable Battery Fast Charge</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frm_battery_protection">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="rad_battery_max_protection">
               <property name="text">
                <string>Maximum Battery Protection (50% charge)</string>
               </property>
               <attribute name="buttonGroup">
                <string>btngrp_battery_protection</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rad_battery_medium_protection">
               <property name="text">
                <string>Medium Battery Protection (80% charge)</string>
               </property>
               <attribute name="buttonGroup">
                <string>btngrp_battery_protection</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rad_battery_no_protection">
               <property name="text">
                <string>No Battery Protection (100% charge)</string>
               </property>
               <attribute name="buttonGroup">
                <string>btngrp_battery_protection</string>
               </attribute>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frm_kbd_bl">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item row="3" column="0">
              <widget class="QLabel" name="lbl_kbd_timeout">
               <property name="text">
                <string>Timeout:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSlider" name="hslider_kbd_timeout">
               <property name="maximum">
                <number>3</number>
               </property>
               <property name="tracking">
                <bool>false</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="lbl_kbd_timeout_val">
               <property name="text">
                <string>10 seconds</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QCheckBox" name="chk_enable_kbd_bl">
               <property name="text">
                <string>Enable Keyboard Backlight</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="chk_enable_touchpad">
            <property name="text">
             <string>Enable Touchpad</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QFrame" name="frm_als">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="chk_enable_als_power">
               <property name="text">
                <string>Enable Ambient Light Sensor</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="lbl_als_lux">
                 <property name="text">
                  <string>Luminance:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="lbl_als_lux_val">
                 <property name="text">
                  <string>ALS Luminance</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="lbl_als_kelvin">
                 <property name="text">
                  <string>Temperature:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="lbl_als_kelvin_val">
                 <property name="text">
                  <string>ALS Temperature</string>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage5" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="frm_lid">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="chk_lid_s3">
               <property name="text">
                <string>Resume from suspend on lid open</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chk_lid_s4">
               <property name="text">
                <string>Resume from hibernate on lid open</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage6" native="true">
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="chk_enable_optdev">
            <property name="text">
             <string>Enable Optical Device</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage7" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QFrame" name="frm_thermal">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="rad_thermal_balanced">
               <property name="text">
                <string>Balanced Profile</string>
               </property>
               <attribute name="buttonGroup">
                <string>btngrp_thermal</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rad_thermal_silent">
               <property name="text">
                <string>Silent Profile</string>
               </property>
               <attribute name="buttonGroup">
                <string>btngrp_thermal</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rad_thermal_performance">
               <property name="text">
                <string>Performance Profile</string>
               </property>
               <attribute name="buttonGroup">
                <string>btngrp_thermal</string>
               </attribute>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btngrp_thermal"/>
  <buttongroup name="btngrp_battery_protection"/>
 </buttongroups>
</ui>
